data/messages.json

```diff
@@ -110,12 +110,19 @@
     "GANGS_NOT_REGISTERED": "@error You need to register on https://sa-mp.nl/ before you can participate in a gang.",
     "GANGS_ONLINE": "{%s}[%s]{FFFFFF} - %s ({B1FC17}%d player%s{FFFFFF})",
 
-    "HOUSE_ANNOUNCE_CREATED": "%s (Id:%d) has created a new house location.",
-    "HOUSE_ANNOUNCE_DELETED": "%s (Id:%d) has deleted a house location (Id:%d).",
-    "HOUSE_ANNOUNCE_EVICTED": "%s (Id:%d) has evicted %s from their house (Id:%d).",
-    "HOUSE_ANNOUNCE_PURCHASED": "%s (Id:%d) has purchased a house for %$.",
-    "HOUSE_ANNOUNCE_SOLD": "%s (Id:%d) has sold the %s house (Id:%d).",
-    "HOUSE_ANNOUNCE_TELEPORTED": "%s (Id:%d) has teleported to one of their houses.",
+    "HOUSE_ANNOUNCE_ACCESS_CHANGED": "(Id:%d) %s has changed (%s (Id:%d))'s access to %s.",
```

Comment:
We use "Nickname (Id:15)" everywhere else, both in-game and on IRC, why change the order?

Reply:
Hey, 

The main reason is: as Beaner requested / mentioned. It'll be a lot easier to search thru the messages if ID is mentioned first as that's not a set value.

Besides after I gave it a thought while speaking with beaner earlier; We're usually more interested in the name and not in the ID. This will let us ignore the ID easier.

We're having ID first on default messages; this lets me (personally) filter/ignore that part of the message as it's often not interesting.

Reply:
Ps. I suggest to change this for all messages.
As it would be a lot of work for less benefit I didn't do them all. We could try to do it as we touch code around it.

Reply:
As discussed; might have the opposite of the wanted effect.

Besides there are IRC bots that parse the messages thus reverted.